integration-tests/hibernate-search-elasticsearch/src/main/java/io/quarkus/test/hibernate/search/elasticsearch/search/DefaultITAnalysisConfigurer.java

```diff
@@ -0,0 +1,13 @@
+package io.quarkus.test.hibernate.search.elasticsearch.search;
+
+import org.hibernate.search.backend.elasticsearch.analysis.ElasticsearchAnalysisConfigurer;
+import org.hibernate.search.backend.elasticsearch.analysis.model.dsl.ElasticsearchAnalysisDefinitionContainerContext;
+
+public class DefaultITAnalysisConfigurer implements ElasticsearchAnalysisConfigurer {
+
+    @Override
+    public void configure(ElasticsearchAnalysisDefinitionContainerContext context) {
+        context.analyzer("standard").type("standard");
```

Comment:
This is not strictly necessary, as the "standard" analyzer is available by default.

Reply:
Same here. It's there to have an analysis configurer and test it's properly handled in the GraalVM native executable.